app/bundles/CoreBundle/Test/EventListener/MaintenanceSubscriberTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * @copyright   2018 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Test\EventListener;
+
+class MaintenanceSubscriberTest extends \PHPUnit_Framework_TestCase
```

Comment:
What's this class for?

Reply:
For testing, but doesn't contain any code :-)
I'll push fix.

Reply:
Fixed